Documentation/Profiling/Profiler Breaking Changes.md

```diff
@@ -0,0 +1,6 @@
+# Profiler Breaking Changes #
+
+Over time we will need to modify the Profiler APIs, this document will serve as a record of any breaking changes.
+
+1. Code Versioning introduced changes documented [here](../design-docs/code-versioning-profiler-breaking-changes.md)
+2. The work to allow adding new types and methods after module load means ICorProfilerInfo7::ApplyMetadata will now potentially trigger a GC, and will not be callable in situations where a GC can not happen (for example  ICorProfilerCallback::RootReferences).
```

Comment:
Profiler vendors may ask us to explain in more detail, but I'm fine keeping this lightweight as you have it now and waiting for them to ask.

Reply:
When you ping the profiler announce thread, I would encourage them to reach out to us if they have questions and concerns though. I think as long as we are open everyone will be happy.